cmd/flow-dps-client/README.md

```diff
@@ -0,0 +1,25 @@
+# Flow DPS Client
+
+## Description
+
+The Flow DPS Client allows access to a Flow DPS Server's index throught the command line. 
+It can be used to execute a Cadence script at an arbitrary block height of a fork.
```

Comment:
Saying `any arbitrary` is like saying `big great`, so that's actually bad style :P

Reply:
I thought the same thing, but it still sounded better to me somehow than the original sentence :p My reasoning is that it feels weird for me to have `a [something] at a [something] of a [something]`.

WDYT of something like this `It can be used to execute Cadence scripts at arbitrary block heights of forks.` ?

Reply:
I think there is nothing at all wrong with that sentence, but I already changed it to `Cadence scripts`.